ui/cypress/e2e/buckets.test.ts

```diff
@@ -293,31 +293,97 @@ describe('Buckets', () => {
 
         cy.getByTestID('bucket-add-line-protocol').click()
         cy.getByTestID('Enter Manually').click()
+        cy.getByTestID('lp-write-data--button').should('be.disabled')
         cy.getByTestID('line-protocol--text-area').type('m1,t1=v1 v=1.0')
-        cy.getByTestID('next').click()
-        cy.getByTestID('line-protocol--status').should('have.class', 'success')
-        cy.getByTestID('next').click()
+        cy.getByTestID('lp-write-data--button').click()
+        cy.getByTestID('line-protocol--status').contains('Success')
+        cy.getByTestID('lp-close--button').click()
 
         // writing a poorly-formed line errors
         cy.getByTestID('add-data--button').click()
         cy.getByTestID('bucket-add-line-protocol').click()
         cy.getByTestID('Enter Manually').click()
         cy.getByTestID('line-protocol--text-area').type('invalid invalid')
-        cy.getByTestID('next').click()
-        cy.getByTestID('line-protocol--status').should('have.class', 'error')
-        cy.getByTestID('next').click()
+        cy.getByTestID('lp-write-data--button').click()
+        cy.getByTestID('line-protocol--status').contains('Unable')
+        cy.getByTestID('lp-cancel--button').click()
 
         // writing a well-formed line with millisecond precision is accepted
-        cy.getByTestID('add-data--button').click()
-        cy.getByTestID('bucket-add-line-protocol').click()
-        cy.getByTestID('Enter Manually').click()
         cy.getByTestID('wizard-step--lp-precision--dropdown').click()
         cy.getByTestID('wizard-step--lp-precision-ms').click()
         const now = Date.now()
         cy.getByTestID('line-protocol--text-area').type(`m2,t2=v2 v=2.0 ${now}`)
-        cy.getByTestID('next').click()
-        cy.getByTestID('line-protocol--status').should('have.class', 'success')
-        cy.getByTestID('next').click()
+        cy.getByTestID('lp-write-data--button').click()
+        cy.getByTestID('line-protocol--status').contains('Success')
+      })
+    })
+
+    it('upload a file and write data', () => {
+      cy.getByTestID('add-data--button').click()
+      cy.getByTestID('bucket-add-line-protocol').click()
+      cy.getByTestID('Upload File').click()
+
+      // When a file is larger than 10MB
+      const bigFile = 'data-big.txt'
```

Comment:
instead of sending in a big file that we gotta tote around, why not make that max size configurable for tests? or use a function to generate it?